tools/profiler/profiler-process/construct_trace.py

```diff
@@ -251,42 +256,38 @@ def enddefinitions(self, vcd, signals, cur_sig_vals):
             k: v for k, v in control_signal_id_to_names.items() if len(v) > 0
         }
 
-    """
-    Reading through value changes and preserving timestamps to value changes for
-    all signals we "care about".
-    """
-
     def value(self, vcd, time, value, identifier_code, cur_sig_vals):
```

Comment:
seems like the `vcd` and `cur_sig_vals` are never used. Not sure if they're for
future work but figured I'd mention!

Reply:
Noted in comments that these variables are unused and only exist bc we inherit the function from `VCDConverter`! :)